drush/Commands/SiteInstallCommands.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\cohesion\Drush\DX8CommandHelpers;
+
+/**
+ * Rebuild site studio when site is installed via drush.
+ */
+class SiteInstallCommands extends DrushCommands {
+
+  /**
+   * Do a forceful rebuild whenever site is installed.
```

Comment:
@saurabhtripathics - comment could be - "Execute site studio rebuild after Acquia CMS site install."

Reply:
changed